tools/system_libs.py

```diff
@@ -1361,10 +1361,10 @@ class libwasmfs(MTLibrary):
   def get_files(self):
     return files_in_path(
         path='system/lib/wasmfs',
-        filenames=['syscalls.cpp', 'file_table.cpp', 'file.cpp', 'wasmfs.cpp', 'streams.cpp', 'memory_file.cpp', 'memory_file_backend.cpp', 'js_file_backend.cpp'])
+        filenames=['syscalls.cpp', 'file_table.cpp', 'file.cpp', 'wasmfs.cpp', 'streams.cpp', 'memory_file.cpp', 'memory_file_backend.cpp', 'js_file_backend.cpp', 'proxied_file_backend.cpp'])
 
   def can_build(self):
-    return settings.WASMFS
+    return True or settings.WASMFS
```

Comment:
@sbc100 When I run the circle CI tests it runs with `embuilder build ALL`. However it seems that `libwasmfs-mt` is not included in `get_system_tasks()` due to the `can_build` being contingent on `settings.WASMFS`.

Is there a good way to include `libwasmfs` when we try to run with `embuilder build ALL`?

Could I maybe just append it to to the tasks in this block?

```
elif 'ALL' in tasks:
    tasks = system_tasks + PORTS
    auto_tasks = True
```

```Exception: FROZEN_CACHE is set, but cache file is missing: "sysroot/lib/wasm32-emscripten/libwasmfs-mt.a" (in cache root path "/root/cache")```